lib/widgets/features/inbox/screens/inbox_invitation_detail.dart

```diff
@@ -208,7 +218,18 @@ class _InboxInvitationDetailScreenState
     AppLocalizations l10n,
     String senderId,
     String? senderName,
+    int selectedReason,
   ) {
+    final AppLocalizations l10n = AppLocalizations.of(context)!;
+
+    List<String> reasons = [
```

Comment:
We'll have to use different values that you don't have access to yet, Saroj. Let me prepare that.

Reply:
I realized `declineChannelInvitation` does hot accept a `reason`, so I will add this it.